articles/curve-fitting-using-matlab/index.md

```diff
@@ -0,0 +1,217 @@
+maximum of 50 or 60 degrees. Accordingly, you can give abound for your data so that Matlab will not consider the other solutions for this parameter. For example,
+if you measure modulus or stiffness parameter, you know that it cannot be negative. It shows the importance of understanding the bound of your given data. 
+So, accordingly, we can choose our coefficients here.
+
+![fit options](curve_seven.png)
+
+By default, it is specified as `-inf`, which is negative of infinity for the lower bound to `inf`, which is positive for the upper bound. So let's say, for
+example, we are a given data, and the law of the data states that the coefficients should vary from 0-1, then we will specify that range in the fit option
+window shown above.
+Specifying the range is essential because it searches for the solutions whose coefficients are within that range alone. In most cases, we have to specify 
+this for the correct solution to our problem. So let's give our range to be o - 1 for the lower limits as shown below;
+
+![lower range](curve_eight.png)
+
+If you want to input your equation, you will choose `custom equation` as the method of fit in the method of fitting. When you select a `custom equation`, 
+a new window opens up that allows you to input your equation.
+
+![input your equation](curve_nine.png)
+
+As we can see, our dependent variable is `y`, and the independent variable is `x`. This dependent variable can be anything depending on the data given. It 
+is why we specify `y` as the function of its independent variables.
+Since our fit is in the form of `y =mx + c` which is given as `a*exp(-b*x)+c`. Assuming that we want our curve to be fitted in the form of `y = mx` and see 
+the results. It means the equation will be `a*x`, and the output is:
+
+![output of equation](curve_ten.png)
+
+Since the autofit is activated, the fit is automatically fitted. As we can see, the curve is not correctly fitted since the constant term is missing. Also, 
+in the result section, we can see more information about the curve. We see it has specified the coefficient as `2.364bound` and the bound as `1.471`, for lower
+limit `3.256` for the upper limit. The goodness of the fit is also given by the error measures as shown. Additionally, we have a warning of a better fitting
+method for our data. It is a suggestion to improve your model.
+
+### How to compare fits
+In this section, now we want to compare two different fits. It involves comparing the coefficients, bounds, and even the goodness of the fits. We want to do 
+this by keeping the fits in a single space and view them. Then, to add a new fit, click on the `fit` menu bar and click on the `new fit` fit option. Once this 
+is done, we see a new window opens up with an autogenerated name `untitled2`. Also, when we look at the `table of fit` section, we see all the information
+about the curve added there.
+
+![new fit](curve_eleven.png)
```

Comment:
Update the image URLs. It will be /engineering-education/name of the folder/ image name.type